doc/debugger.rst

```diff
@@ -1773,6 +1808,74 @@ target endianness and interpret the bytecode based on that.
 .. note:: This command is somewhat incomplete at the moment and may be modified
    once the best way to do this in the debugger UI has been figured out.
 
+AppRequest request (0x23)
+-------------------------
+
+Format::
+
+    REQ <int: 0x23> [<tval>*] EOM
+    REP [<tval>*] EOM
+
+Example::
+
+    REQ 35 "GameInfo" "GetTitle" EOM
+    REP "Spectacles: Bruce's Story" EOM
+
+This is a custom request message whose meaning and semantics depend on the
+application.
+
+AppRequest messages are used for direct communication between the debug client
+and debug target over the Duktape debug protocol. Both the meaning of a custom
+message and the dvalues it contains are entirely up to the implementation and
```

Comment:
Minuscule convention consistency issue: I've used two spaces after a period (`foo bar.  Quux...`) in the RST documents. I'm happy to fix this very trivial issue in a separate pull though :)